site/_layouts/post.html

```diff
@@ -0,0 +1,80 @@
+<!-- Blog Layout Start -->
+{% assign abc = page.author | split: "," %}
+{% if abc.size > 1 %}
+{% assign img_link = abc.last %}
+{% else %}
+{% assign img_link = "http://pageanthive.com/images/empty-profile.png" %}
+{% endif %}
+{% assign name = abc.first %}
+<!DOCTYPE html>
+<html lang="{{ site.lang }}">
+
+  {% include head.html %}
+
+  <body id="page-top" data-spy="scroll" data-target=".navbar"  data-offset="151">
+
+    {% include navigation.html %}
+    {% if page.title == 'Blog' %}
+    <div class="container-fluid" id="topCont">
+      <div class="row">
+        <img src="{{ 'assets/images/blogTop.png' | relative_url }}">
+      </div>
+    </div>
+    {% endif %}
+
+    <div class="container" id="postsContainer">
+      <div class="row">
+    {% if page.title != 'Blog' %}
+      <div class="col-md-10 col-lg-10 col-md-offset-1 col-lg-offset-1">
+        <div class="row blogPost">
+          <div class="col-md-12 col-lg-12">
+            <div class="row postTop">
+              <p class="timeStamp">{{ name }} blogged on {{ page.created_at | date: "%a, %d %b %Y" }}</p>
+              <img class="postWriter" src="{{ img_link }}" style="min-width:150px; min-height:150px; max-height:150px; max-width:150px;">
+              <p class="postTitle">{{ page.title }}</p>
+            </div>
+            <hr>
+            <div class="row postContent">
+              <p>
+                {{ page.content }}
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+    {% else %}
+      {% if site.posts.size == 0 %}
+        <h2>No blogs have been added until now. Stay tuned !</h2>
+      {% endif %}
+      {% for post in site.posts %}
+        <div class="col-md-10 col-lg-10 col-md-offset-1 col-lg-offset-1">
+          <div class="row blogPost">
+            <div class="col-md-12 col-lg-12">
+              <div class="row postTop">
+                {% assign authp = post.author | split: "," %}
+                <p class="timeStamp">{{ authp.first }} blogged on {{ post.created_at | date: "%a, %d %b %Y" }}</p>
+                <img class="postWriter" src="{{ authp[1] | default: 'http://pageanthive.com/images/empty-profile.png' }}" style="min-width:150px; min-height:150px; max-height:150px; max-width:150px;">
```

Comment:
Why do we have the link to empty profile both here and in the front matter?

Reply:
Will try debugging a little more, 
Sheer laziness I guess xD